portfolio/src/main/client/src/components/controls/lightbox.ts

```diff
@@ -45,5 +46,9 @@ const lightbox: HTMLDivElement = htmlElement`
   </div>`;
 
 lightbox.addEventListener('click', hideLightbox);
+window.addEventListener('keydown', (e) => {
```

Comment:
nit: add a comment that this closes the lightbox on Escape/Space being pressed. Or you can store the keycode as a const:
```
const escapeKey = 27;
if (e.keyCode !== escapeKey); return;
```